test/unit/metadata_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'unit_helper'
+
+require 'mountable_file_server/metadata'
+
+class MetadataTest < UnitTestCase
+  Metadata = MountableFileServer::Metadata
+
+  def test_size
+    subject = Metadata.for_path(fixture_path('image.png'))
+    result = subject.size
+    assert_equal File.size(fixture_path('image.png')), result
+  end
+
+  def test_content_type
+    subject = Metadata.for_path(fixture_path('image.png'))
+    result = subject.content_type
+    assert_equal 'image/png', result
+  end
+
+  def test_image_height
+    subject = Metadata.for_path(fixture_path('image.png'))
+    result = subject.height
+    assert_equal 62, result
+  end
+
+  def test_image_width
+    subject = Metadata.for_path(fixture_path('image.png'))
+    result = subject.width
+    assert_equal 62, result
```

Comment:
Using a square image it probably not ideal when i comes to verifying the dimensions.